### PR TITLE
Circleci project setup

### LINKED ISSUE #2768: Integrate Circle CI to automate the APK build
Push build script for CircleCI

**Acceptance criteria**
1. Build should trigger for pull requests only
2. Circle CI should build the debug APK only (Fdroid and Play Store)
3. APK link needs to be commented out on the PR itself

### PR DESCRIPTION
**Pull request by spacemonkey1101**
Fixes #2768

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -1,5 +1,5 @@
 plugins {
-    id "com.diffplug.gradle.spotless" version "3.26.1"
+    id "com.diffplug.gradle.spotless" version "3.27.0"
```
Comment (liveHarshit) [inline app/build.gradle:2]: revert this

**Review by liveHarshit — COMMENTED** [review]
@iamanbansal Can you please add circle ci to this repo?

**Comment by iamanbansal** [comment]
@liveHarshit  I think you might tagged the wrong person :D

**Comment by liveHarshit** [comment]
> @liveHarshit I think you might tagged the wrong person :D

sorry, @iamareebjamal  Can you please add circle ci to this repo?

**Pull request closed without merging**